components/filterPane.jsx

```diff
@@ -0,0 +1,28 @@
+// TODO: this isn't being imported in pages/index.jsx yet! Figure out how to make this useful and then do that
+import React from "react";
+
+const FilterPane = () => {
+  return (
+    <div className="filter-pane white-and-blue">
+      <div className="legend" id="key"></div>
+      <button
```

Comment:
Should these be the `HelpToggle` and `LocationListToggle` button components?

Reply:
Oooh yep definitely! I haven't worked on this component at all, pretty much just copied the existing markup over. This component and all of its logic (i.e. the search box and check boxes) are like the main outstanding pieces for this PR!